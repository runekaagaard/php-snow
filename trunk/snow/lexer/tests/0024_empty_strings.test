call_me("")
call_you("""""")
call_us('')
call_her('''''')
echo "{"x"}"
echo ""
----
NAME: call_me
LPAR: (
STRING: 
RPAR: )
NEWLINE: \n
NAME: call_you
LPAR: (
STRING: 
RPAR: )
NEWLINE: \n
NAME: call_us
LPAR: (
STRING: 
RPAR: )
NEWLINE: \n
NAME: call_her
LPAR: (
STRING: 
RPAR: )
NEWLINE: \n
ECHO: echo
STRING: x
NEWLINE: \n
ECHO: echo
STRING: 
ENDMARKER: None