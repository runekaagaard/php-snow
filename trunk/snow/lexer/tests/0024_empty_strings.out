NAME: call_me
LPAR: (
RPAR: )
NEWLINE: \n
NAME: call_you
LPAR: (
RPAR: )
NEWLINE: \n
NAME: call_us
LPAR: (
RPAR: )
NEWLINE: \n
NAME: call_her
LPAR: (
RPAR: )
NEWLINE: \n
ECHO: echo
STRING: x
NEWLINE: \n
ECHO: echo
ENDMARKER: None